test/parallel/test-util-inspect.js

```diff
@@ -474,6 +474,7 @@ test_lines({
 
 // test boxed primitives output the correct values
 assert.equal(util.inspect(new String('test')), '[String: \'test\']');
+assert.equal(util.inspect(Object(Symbol('test'))), '[Symbol: Symbol(test)]');
```

Comment:
@thefourtheye I think learning the difference between `Symbol(test)` and `[Symbol: test]` might be a bit harder for people new to symbols.

This way might not be perfect, but I think the consistency with how other boxed values are displayed makes it easier to reason about what exactly is being displayed.